ciphers/onepad_cipher.py

```diff
@@ -4,7 +4,17 @@
 class Onepad:
     @staticmethod
     def encrypt(text: str) -> tuple[list[int], list[int]]:
-        """Function to encrypt text using pseudo-random numbers"""
+        """
+        Function to encrypt text using pseudo-random numbers
+        >>> Onepad().encrypt("")
+        ([], [])
+        >>> random.seed(1)
+        >>> Onepad().encrypt(" ")
+        ([6969], [69])
+        >>> random.seed(1)
+        >>> Onepad().encrypt("Hello")
+        ([9729, 114756, 4653, 31309, 10492], [69, 292, 33, 131, 61])
```

Comment:
Let's make it raise some Exceptions:
```suggestion
        ([9729, 114756, 4653, 31309, 10492], [69, 292, 33, 131, 61])
        >>> Onepad().encrypt(1)
        >>> Onepad().encrypt(1.1)
        >>> Onepad().encrypt([])
```